packages/db-dashboard/package.json

```diff
@@ -23,29 +23,33 @@
     "lint": "standard | snazzy"
   },
   "dependencies": {
-    "@fastify/static": "^6.5.0"
+    "@fastify/static": "^6.5.0",
+    "@mui/material": "^5.10.11",
+    "@platformatic/db-ra-data-rest": "workspace:*",
+    "inflected": "^2.1.0",
+    "react-admin": "^4.4.4"
   },
   "devDependencies": {
     "@graphiql/toolkit": "^0.8.0",
     "@playwright/test": "^1.24.2",
+    "@types/react": "^18.0.17",
+    "@types/react-dom": "^18.0.6",
```

Comment:
Hey there, I think we should remove these two packages. They should not be needed since we're not using typescript

Reply:
done